api/index.md

```diff
@@ -33,4 +32,35 @@ The endpoints can be roughly grouped into four categories:
     graph query language [SPARQL](https://www.w3.org/TR/rdf-sparql-query/){: target="_blank"}. This is useful for complex node connections which would require multiple API calls; for example, "hate crimes motivated by disability status in Californian cities".
 
 -   **Utilities**: These are Python notebook-specific APIs for helping with
-    Pandas DataFrames, etc.
\ No newline at end of file
+    Pandas DataFrames, etc.
+
+In addition, Data Commons provides additional tools for accessing its data that call the REST APIs under the hood:
+
+- [Google Sheets](sheets/index.md): provides several custom functions that populate spreadsheets with data from the Data Commons knowledge graph
+- [Web Components](web_components/index.md): provides JavaScript APIs and HTML templates that allow you to embed Data Commons data and visualizations into web pages
+
+
+## API keys {: #get-key}
+
+A key is required by some APIs to authenticate and authorize requests. 
+- All REST [V2](rest/v2/index.md) and [V1](rest/v1/index.md) APIs. These requests are served by endpoints at `api.datacommons.org`.
+- All requests coming from a custom Data Commons instance. These are also served by `api.datacommons.org`.
+- All DataGemma APIs <!-- TODO: Add link to docs about these when available -->. These requests are served by endpoints at `datagemma.datacommons.org`.
+
+A key is currently not required for the following, although this may change in the future:
+- Python and Pandas client libraries other than DataGemma
+- V0 REST APIs
+- Google Sheets
+- Web Components
+
+### Obtain an API key 
+
+Data Commons API keys are managed by Apigee. To obtain an API key, go to [https://apikeys.datacommons.org](https://apikeys.datacommons.org){: target="_blank"}  and request a key for the hostname(s) listed above. You will need separate keys for Data Commons and DataGemma.
```

Comment:
IMO we should not mention Apigee directly since it is an implementation detail. Maybe "Data Commons API keys are managed through a self-service portal."

Reply:
OK, I didn't know if users might need to interact with Apigee in any way, but if they don't, then I agree.